### DISCUSSION TITLE
Production to heroku

### DISCUSSION THREAD

**Discussion by ghost** [adonisjs/core#3790] [Help]
Hello everyone, I've build a small application but when I deploy it to heroku the app is deployed successfully but I got error when I lunch it, heroku log --tail
![IMG_20220708_153254_037](https://user-images.githubusercontent.com/78963810/178093671-544b8a25-8248-4e22-b519-10619fb60102.jpg)

**Comment by Barbapapazes** [comment]
Hello, 

Can you provide more details? (a reproduction repo?) Thanks!

**Reply by Barbapapazes** [reply]
Answer could be here: https://github.com/adonisjs/core/discussions/3309#discussioncomment-3116049

**Reply by ghost** [reply]
Hello, thanks I've solved the problem I'll provide a medium article which I'll write. Thanks

**Reply by Barbapapazes** [reply]
Don't hesitate to publish the link here for others! Have a great day! ✨
